src/components/dropdown/dropdown.ts

```diff
@@ -66,6 +66,7 @@ export class MDropdown extends BaseDropdown implements MDropdownInterface {
     private internalFilterRegExp: RegExp = / /;
 
     private internalItems: MDropdownItem[] = [];
+    private internalIsFocus: boolean = false;
```

Comment:
Je renommerais peut-être la variable `internalIsFocus` par 'isFocus`

Reply:
Updated it ;). Merci